packages/bbui/src/Form/Core/DatePicker/Calendar.svelte

```diff
@@ -4,12 +4,10 @@
   import dayjs from "dayjs"
   import NumberInput from "./NumberInput.svelte"
   import { createEventDispatcher } from "svelte"
-  import isoWeek from "dayjs/plugin/isoWeek"
   import Icon from "../../../Icon/Icon.svelte"
 
-  dayjs.extend(isoWeek)
-
   export let value
+  export let startDayOfWeek = "Monday"
```

Comment:
Depending on what country you are in, the start of the week isn't always Monday.

Reply:
Added `getLocaleStartDayOfWeek` which takes the first day of the week from the browser locale, but can still be overridden.